### ISSUE TITLE
Battle.net closes after launch

### ISSUE THREAD

**Issue by superbonaci** [lutris/lutris#2408]
After latest update, on ubuntu:
```
Running /home/myuser/.local/share/lutris/runners/wine/lutris-4.13-x86_64/bin/wine /home/myuser/Games/overwatch/drive_c/Program Files (x86)/Battle.net/Battle.net.exe --exec=launch Pro
esync: up and running.
[1016/184642.687:ERROR:network_change_notifier_win.cc(156)] WSALookupServiceBegin failed with: 0
[1016/184643.628:ERROR:mf_helpers.cc(14)] Error in dxva_video_decode_accelerator_win.cc on line 373
info:  Game: Battle.net.exe
info:  DXVK: v1.4.2
info:  Found config file: /home/myuser/Games/overwatch/dxvk.conf
info:  Effective configuration:
info:    dxgi.nvapiHack = False
warn:  OpenVR: Failed to locate module
info:  Enabled instance extensions:
info:    VK_KHR_get_physical_device_properties2
info:    VK_KHR_surface
info:    VK_KHR_win32_surface
info:  GeForce GTX 1080:
info:    Driver: 418.56.0
info:    Vulkan: 1.1.95
info:    Memory Heap[0]: 
info:      Size: 8192 MiB
info:      Flags: 0x1
info:      Memory Type[7]: Property Flags = 0x1
info:      Memory Type[8]: Property Flags = 0x1
info:    Memory Heap[1]: 
info:      Size: 11992 MiB
info:      Flags: 0x0
info:      Memory Type[0]: Property Flags = 0x0
info:      Memory Type[1]: Property Flags = 0x0
info:      Memory Type[2]: Property Flags = 0x0
info:      Memory Type[3]: Property Flags = 0x0
info:      Memory Type[4]: Property Flags = 0x0
info:      Memory Type[5]: Property Flags = 0x0
info:      Memory Type[6]: Property Flags = 0x0
info:      Memory Type[9]: Property Flags = 0x6
info:      Memory Type[10]: Property Flags = 0xe
info:  D3D11CoreCreateDevice: Probing D3D_FEATURE_LEVEL_11_1
info:  D3D11CoreCreateDevice: Using feature level D3D_FEATURE_LEVEL_11_1
info:  Device properties:
info:    Device name:     : GeForce GTX 1080
info:    Driver version   : 418.56.0
info:  Enabled device extensions:
info:    VK_EXT_conditional_rendering
info:    VK_EXT_shader_viewport_index_layer
info:    VK_EXT_transform_feedback
info:    VK_EXT_vertex_attribute_divisor
info:    VK_KHR_create_renderpass2
info:    VK_KHR_dedicated_allocation
info:    VK_KHR_depth_stencil_resolve
info:    VK_KHR_descriptor_update_template
info:    VK_KHR_draw_indirect_count
info:    VK_KHR_driver_properties
info:    VK_KHR_get_memory_requirements2
info:    VK_KHR_image_format_list
info:    VK_KHR_maintenance1
info:    VK_KHR_maintenance2
info:    VK_KHR_sampler_mirror_clamp_to_edge
info:    VK_KHR_shader_draw_parameters
info:    VK_KHR_swapchain
info:  Device features:
info:    robustBufferAccess                     : 1
info:    fullDrawIndexUint32                    : 1
info:    imageCubeArray                         : 1
info:    independentBlend                       : 1
info:    geometryShader                         : 1
info:    tessellationShader                     : 1
info:    sampleRateShading                      : 1
info:    dualSrcBlend                           : 1
info:    logicOp                                : 1
info:    multiDrawIndirect                      : 1
info:    drawIndirectFirstInstance              : 1
info:    depthClamp                             : 1
info:    depthBiasClamp                         : 1
info:    fillModeNonSolid                       : 1
info:    depthBounds                            : 1
info:    multiViewport                          : 1
info:    samplerAnisotropy                      : 1
info:    textureCompressionBC                   : 1
info:    occlusionQueryPrecise                  : 1
info:    pipelineStatisticsQuery                : 1
info:    vertexPipelineStoresAndAtomics         : 1
info:    fragmentStoresAndAtomics               : 1
info:    shaderImageGatherExtended              : 1
info:    shaderStorageImageExtendedFormats      : 1
info:    shaderStorageImageReadWithoutFormat    : 1
info:    shaderStorageImageWriteWithoutFormat   : 1
info:    shaderClipDistance                     : 1
info:    shaderCullDistance                     : 1
info:    shaderFloat64                          : 1
info:    shaderInt64                            : 1
info:    variableMultisampleRate                : 1
info:  VK_EXT_conditional_rendering
info:    conditionalRendering                   : 1
info:  VK_EXT_depth_clip_enable
info:    depthClipEnable                        : 0
info:  VK_EXT_host_query_reset
info:    hostQueryReset                         : 0
info:  VK_EXT_memory_priority
info:    memoryPriority                         : 0
info:  VK_EXT_shader_demote_to_helper_invocation
info:    shaderDemoteToHelperInvocation         : 0
info:  VK_EXT_transform_feedback
info:    transformFeedback                      : 1
info:    geometryStreams                        : 1
info:  VK_EXT_vertex_attribute_divisor
info:    vertexAttributeInstanceRateDivisor     : 1
info:    vertexAttributeInstanceRateZeroDivisor : 1
info:  Queue families:
info:    Graphics : 0
info:    Transfer : 1
info:  DXVK: Read 27 valid state cache entries
info:  DXVK: Using 4 compiler threads
info:  Adapter LUID 0: 0:ae1
err:   D3D11DXGIResource::GetSharedHandle: Stub
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  9d8e1289-d7b3-465f-8126-250e349af85d
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR
info:    Buffer size:  104x1
info:    Image count:  2
warn:  DXGI: MakeWindowAssociation: Ignoring flags
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  9d8e1289-d7b3-465f-8126-250e349af85d
err:   D3D11DXGIResource::GetSharedHandle: Stub
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  9d8e1289-d7b3-465f-8126-250e349af85d
info:  Presenter: Actual swap chain properties:
info:    Format:       VK_FORMAT_B8G8R8A8_UNORM
info:    Present mode: VK_PRESENT_MODE_IMMEDIATE_KHR
info:    Buffer size:  1200x756
info:    Image count:  2
warn:  DXGI: MakeWindowAssociation: Ignoring flags
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  9d8e1289-d7b3-465f-8126-250e349af85d
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  f8fb5c27-c6b3-4f75-a4c8-439af2ef564c
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  f8fb5c27-c6b3-4f75-a4c8-439af2ef564c
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  f8fb5c27-c6b3-4f75-a4c8-439af2ef564c
warn:  D3D11Texture2D::QueryInterface: Unknown interface query
warn:  f8fb5c27-c6b3-4f75-a4c8-439af2ef564c
wine: Call from 0x7b452f1c to unimplemented function api-ms-win-crt-private-l1-1-0.dll._o_getenv, aborting
wine: Call from 0x7b452f1c to unimplemented function api-ms-win-crt-private-l1-1-0.dll._o_getenv, aborting
wine: Unimplemented function api-ms-win-crt-private-l1-1-0.dll._o_getenv called at address 0x7b452f1c (thread 00b6), starting debugger...
Unhandled exception: unimplemented function api-ms-win-crt-private-l1-1-0.d._o_getenv called in 32-bit code (0x7b452f1c).
Register dump:
 CS:0023 SS:002b DS:002b ES:002b FS:0063 GS:006b
 EIP:7b452f1c ESP:0f9cf814 EBP:0f9cf888 EFLAGS:00000216(   - --  I   -A-P- )
 EAX:0f9cf82c EBX:eed96902 ECX:00000004 EDX:0f9cf8b4
 ESI:0f9cfdc4 EDI:0f9cfaac
Stack dump:
0x0f9cf814:  00000000 00000000 00000000 00000000
0x0f9cf824:  00000000 00000000 80000100 00000001
0x0f9cf834:  00000000 7b452f1c 00000002 eed96902
0x0f9cf844:  eed99c0c 0d70a1b0 00000018 00000000
0x0f9cf854:  00000000 00000000 4d430003 00000000
0x0f9cf864:  00000000 00000000 00000000 00000000
Backtrace:
=>0 0x7b452f1c in kernel32 (+0x22f1c) (0x0f9cf888)
  1 0xeed9671f (0x0f9cf8c8)
  2 0xeed94f90 (0x0f9cf8f4)
  3 0x117ad7aa EntryPoint+0xffffffff() in libglesv2 (0x0f9cfb2c)
  4 0x11794b8a EntryPoint+0xffffffff() in libglesv2 (0x0f9cfcb8)
  5 0x117a9de5 EntryPoint+0xffffffff() in libglesv2 (0x0f9cfde0)
  6 0x117abf03 EntryPoint+0xffffffff() in libglesv2 (0x0f9cfe18)
  7 0x11765f47 EntryPoint+0xffffffff() in libglesv2 (0x0f9cfe20)
  8 0x117674ff EntryPoint+0xffffffff() in libglesv2 (0x0f9cfe64)
  9 0x117680b6 EntryPoint+0xffffffff() in libglesv2 (0x0f9cfe78)
  10 0x11767015 EntryPoint+0xffffffff() in libglesv2 (0x0f9cfe88)
  11 0x118e7d1f EntryPoint+0xffffffff() in libglesv2 (0x0f9cfea8)
  12 0x7bcdf828 (0x0f9cff38)
  13 0x7bcce734 (0x0f9cff4c)
  14 0x7bcd230e (0x0f9cffdc)
  15 0x7bcce726 (0x0f9cffec)
0x7b452f1c: addl	$12,%esp
Modules:
Module	Address			Debug info	Name (136 modules)
PE	  370000-  3f3000	Deferred        qt5winextras
PE	  400000-  514000	Deferred        battle.net
PE	  740000-  c12000	Deferred        qt5gui
PE	  c20000-  cbd000	Deferred        qt5multimedia
PE	  cc0000-  fb9000	Deferred        qt5quick
PE	  fc0000- 14f2000	Deferred        vivoxsdk
PE	 1500000- 158a000	Deferred        ortp
PE	 1590000- 729c000	Deferred        libcef
PE	 72a0000- 7365000	Deferred        chrome_elf
PE	 7b20000- 7c3d000	Deferred        qwindows
PE	 7d50000- 7d5b000	Deferred        qgif
PE	 7d60000- 7d6c000	Deferred        qico
PE	 7d70000- 7db0000	Deferred        qjpeg
PE	 7db0000- 7deb000	Deferred        qmng
PE	 7df0000- 7df9000	Deferred        qsvg
PE	 7e00000- 7e42000	Deferred        qt5svg
PE	 7e50000- 7ea5000	Deferred        qtiff
PE	 7eb0000- 7ef4000	Deferred        bzrecord
PE	 a070000- a21a000	Deferred        fmod
PE	 e270000- e290000	Deferred        libegl
PE	 e290000- e299000	Deferred        qtquick2plugin
PE	 e2e0000- e2e9000	Deferred        qtgraphicaleffectsplugin
PE	 e2f0000- e2ff000	Deferred        qtgraphicaleffectsprivate
PE	 ff40000- ff55000	Deferred        qquicklayoutsplugin
PE	 ff60000- ff69000	Deferred        windowplugin
PE	 ffd0000- ffea000	Deferred        qtquickcontrols2plugin
PE	10000000-11666000	Deferred        battle.net
PE	11670000-11de1000	Export          libglesv2
PE	1a810000-1a82b000	Deferred        qt5quickcontrols2
PE	1a830000-1a8da000	Deferred        qt5quicktemplates2
PE	1a920000-1a966000	Deferred        qtquickcontrolsplugin
PE	1a970000-1a9a8000	Deferred        qtquicktemplates2plugin
PE	1b0d0000-1b0d9000	Deferred        modelsplugin
PE	1c250000-1c5e0000	Deferred        d3dcompiler_47
PE	61000000-6102b000	Deferred        qt5xml
PE	64000000-640dc000	Deferred        qt5network
PE	65000000-65486000	Deferred        qt5widgets
PE	66000000-662d5000	Deferred        qt5qml
PE	67000000-674bf000	Deferred        qt5core
PE	6a540000-6a823000	Deferred        d3d11
PE	6f400000-6f5f0000	Deferred        dxgi
PE	7a840000-7a849000	Deferred        opengl32
PE	7b430000-7b603000	Export          kernel32
PE	7bc30000-7bc47000	Deferred        ntdll
PE	7c8d0000-7c8d6000	Deferred        uxtheme
PE	7ccf0000-7cd09000	Deferred        setupapi
PE	7ced0000-7ced5000	Deferred        winex11
PE	7cf90000-7cfd8000	Deferred        crypt32
PE	7d090000-7d98b000	Deferred        shell32
PE	7db50000-7db55000	Deferred        bcrypt
PE	7db70000-7db77000	Deferred        wintrust
PE	7dbd0000-7dbde000	Deferred        oleaut32
PE	7dcf0000-7dcf7000	Deferred        ws2_32
PE	7dd20000-7dd26000	Deferred        imm32
PE	7dd50000-7dd54000	Deferred        usp10
PE	7ddb0000-7de06000	Deferred        comctl32
PE	7def0000-7defa000	Deferred        aclui
PE	7df20000-7df2d000	Deferred        rpcrt4
PE	7dfd0000-7dffd000	Deferred        ole32
PE	7e110000-7e116000	Deferred        shcore
PE	7e140000-7e152000	Deferred        shlwapi
PE	7e1c0000-7e1d5000	Deferred        msvcrt
PE	7e270000-7e275000	Deferred        version
PE	7e2a0000-7e2ad000	Deferred        advapi32
PE	7e330000-7e33e000	Deferred        gdi32
PE	7e490000-7e584000	Deferred        user32
PE	7e6b0000-7e6bd000	Deferred        mpr
PE	7e700000-7e723000	Deferred        wininet
PE	7ee80000-7ee83000	Deferred        wow64cpu
PE	7eea0000-7eeb8000	Deferred        kernelbase
PE	eed80000-eed90000	Deferred        api-ms-win-crt-private-l1-1-0
PE	f5d50000-f5d55000	Deferred        schannel
PE	f5d70000-f5d75000	Deferred        msctf
PE	f5df0000-f5e61000	Deferred        msvcp120
PE	f5ee0000-f5eea000	Deferred        msacm32
PE	f6010000-f6018000	Deferred        combase
PE	f6210000-f6214000	Deferred        avrt
PE	f6220000-f6225000	Deferred        wbemprox
PE	f62d0000-f62d4000	Deferred        winepulse
PE	f6410000-f6415000	Deferred        mmdevapi
PE	f6b20000-f6b24000	Deferred        kerberos
PE	f6b30000-f6b6b000	Deferred        concrt140
PE	f6b30000-f6b6b000	Deferred        dhcpcsvc
PE	f6b70000-f6b76000	Deferred        ncrypt
PE	f6b80000-f6c5c000	Deferred        cryptui
PE	f6c90000-f6d3d000	Deferred        comdlg32
PE	f6da0000-f6dae000	Deferred        winspool
PE	f6df0000-f6e0d000	Deferred        credui
PE	f6e30000-f6e44000	Deferred        urlmon
PE	f6ed0000-f6ed4000	Deferred        dxva2
PE	f6f30000-f6f3a000	Deferred        wined3d
PE	f7090000-f7095000	Deferred        d3d9
PE	f7130000-f7138000	Deferred        winevulkan
PE	f7170000-f7178000	Deferred        vulkan-1
PE	f71a0000-f71a4000	Deferred        dwrite
PE	f7220000-f7228000	Deferred        propsys
PE	f7240000-f7245000	Deferred        hid
PE	f7260000-f7265000	Deferred        uiautomationcore
PE	f7280000-f7298000	Deferred        oleacc
PE	f72e0000-f72e4000	Deferred        userenv
PE	f7300000-f7305000	Deferred        secur32
PE	f7340000-f7344000	Deferred        sensapi
PE	f7350000-f7356000	Deferred        jsproxy
PE	f7380000-f7387000	Deferred        winhttp
PE	f73d0000-f73d4000	Deferred        wtsapi32
PE	f73e0000-f73e6000	Deferred        dwmapi
PE	f7400000-f7407000	Deferred        iphlpapi
PE	f7430000-f7438000	Deferred        netapi32
PE	f7490000-f7497000	Deferred        dnsapi
PE	f74b0000-f74b3000	Deferred        api-ms-win-crt-locale-l1-1-0
PE	f74d0000-f74dc000	Deferred        dinput8
PE	f7520000-f7525000	Deferred        mswsock
PE	f7560000-f764e000	Deferred        msvcr120
PE	f7560000-f764e000	Deferred        api-ms-win-crt-filesystem-l1-1-0
PE	f7660000-f7664000	Deferred        api-ms-win-crt-convert-l1-1-0
PE	f7670000-f7673000	Deferred        api-ms-win-crt-time-l1-1-0
PE	f7680000-f7684000	Deferred        api-ms-win-crt-string-l1-1-0
PE	f76a0000-f76a5000	Deferred        api-ms-win-crt-math-l1-1-0
PE	f76b0000-f76b3000	Deferred        api-ms-win-crt-heap-l1-1-0
PE	f76d0000-f76d3000	Deferred        api-ms-win-crt-utility-l1-1-0
PE	f76e0000-f76e3000	Deferred        api-ms-win-crt-environment-l1-1-0
PE	f76f0000-f76f4000	Deferred        api-ms-win-crt-stdio-l1-1-0
PE	f7700000-f7704000	Deferred        api-ms-win-crt-runtime-l1-1-0
PE	f7720000-f7734000	Deferred        vcruntime140
PE	f7760000-f7883000	Deferred        ucrtbase
PE	f78a0000-f7910000	Deferred        msvcp140
PE	f7990000-f7a10000	Deferred        winmm
PE	f7a50000-f7a55000	Deferred        rsaenh
PE	f7aa0000-f7aa7000	Deferred        dbghelp
PE	f7b10000-f7b15000	Deferred        imagehlp
PE	f7b30000-f7b34000	Deferred        api-ms-win-core-localization-obsolete-l1-2-0
PE	f7b40000-f7b44000	Deferred        api-ms-win-core-datetime-l1-1-1
PE	f7b60000-f7b64000	Deferred        api-ms-win-core-string-l1-1-0
PE	f7b70000-f7b74000	Deferred        api-ms-win-core-localization-l1-2-1
PE	f7b80000-f7b84000	Deferred        api-ms-win-core-fibers-l1-1-1
PE	f7b90000-f7b94000	Deferred        api-ms-win-core-synch-l1-2-0
Threads:
process  tid      prio (all id:s are in hex)
00000008 (D) C:\Program Files (x86)\Battle.net\Battle.net.exe
	["C:\Program Files (x86)\Battle.net\Battle.net.exe" "--exec=launch Pro"]
	000000ba    0
	000000b7    0
	000000b6    0 <==
	000000b4    0
	000000b3    0
	000000b2    0
	000000b1    0
	000000b0   -2
	000000af   -2
	000000ae   -2
	000000ad   -2
	00000079    0
	00000074    0
	00000073   15
	0000006c    0
	0000006b    0
	00000069    0
	00000068    0
	00000067   -2
	00000064    0
	00000063    0
	00000062    0
	00000061    0
	00000060    0
	0000005f    0
	0000005e    0
	0000005d   -2
	0000005c    0
	0000005b    0
	0000005a    0
	00000059    0
	00000055    0
	00000054    0
	00000053    0
	00000052    0
	00000051    0
	00000050    0
	0000004f    0
	0000004e    0
	0000004d   -2
	0000004c    0
	0000004b    0
	0000004a    0
	00000049    0
	00000048    0
	00000047    0
	00000046    0
	00000045    0
	00000044    0
	00000041    0
	0000003e    0
	0000003d   15
	0000003c    0
	0000003b    0
	0000003a    0
	00000039    0
	00000038    0
	00000037    0
	00000035    0
	00000032    0
	00000031    0
	00000030    0
	0000002f    0
	0000002b    0
	00000009    0
0000000e services.exe
	00000026    0
	00000023    0
	0000001f    0
	0000001c    0
	00000013    0
	00000010    0
	0000000f    0
00000011 plugplay.exe
	00000017    0
	00000016    0
	00000012    0
00000018 explorer.exe
	0000002a    0
	00000029    0
	00000028    0
	00000019    0
0000001a winedevice.exe
	00000020    0
	0000001e    0
	0000001d    0
	0000001b    0
00000021 winedevice.exe
	00000027    0
	00000025    0
	00000024    0
	00000022    0
0000003f Agent.exe
	["C:\ProgramData\Battle.net\Agent\Agent.6847\Agent.exe" --session=4298702758555384299]
	000000a1    0
	000000a0    0
	0000009f    0
	0000009e    0
	0000009d    0
	00000099    0
	00000098    0
	00000097    0
	00000096    0
	00000095    0
	00000094    0
	00000093    0
	00000092    0
	00000091    0
	00000090    0
	0000008f    0
	0000008e    0
	0000007e    0
	00000058    0
	00000056    0
	00000040    0
00000042 wineconsole.exe
	[wineconsole --use-event=20]
	00000043    0
00000065 Battle.net.exe
	["C:\Program Files (x86)\Battle.net\Battle.net.exe" --type=gpu-process --field-trial-handle=980,10361626846685735723,16080900375655771830,131072 --disable-features=HardwareMediaKeyHandling --no-sandbox --log-file="C:\users\myuser\Local Settings\Application Data\Battle.net\Logs\libcef-20191016T164642.570962.log" --log-severity=error --product-version="Battle.net/1.15.5.11555 Chrome/75.0.3770.100" --lang=en-US --watch-browser-pid=8 --gpu-preferences=KAAAAAAAAADgAAAgAQAAAAAAAAAAAGAAAAAAAAAAAAAIAAAAAAAAACgAAAAEAAAAIAAAAAAAAAAoAAAAAAAAADAAAAAAAAAAOAAAAAAAAAAQAAAAAAAAAAAAAAAFAAAAEAAAAAAAAAAAAAAABgAAABAAAAAAAAAAAQAAAAUAAAAQAAAAAAAAAAEAAAAGAAAA --use-gl=swiftshader-webgl --log-file="C:\users\myuser\Local Settings\Application Data\Battle.net\Logs\libcef-20191016T164642.570962.log" --service-request-channel-token=16748612847657221739 --mojo-platform-channel-handle=984 /prefetch:2 --battle-net-helper=Battle.net.11555]
	0000008d    0
	0000008c    0
	0000008b    0
	0000008a    0
	00000089    0
	00000088   -2
	00000087    0
	00000086    0
	00000084    0
	00000083    0
	00000066    0
System information:
    Wine build: wine-4.13.r0.g3f06c18b ( TkG Staging Esync Fsync )
    Platform: i386 (WOW64)
    Version: Windows 10
    Host system: Linux
    Host version: 5.0.0-32-generic
Waiting on children
All children gone
Exit with returncode 1
```

**Comment by G4E** [comment]
Upgrade your runner to at least wine 4.16 and it should go away automagically.

**Comment by AccountOneOff** [comment]
Please, only open issues related to the Lutris client. We have other avenues for game support. Thank you.

**Comment by superbonaci** [comment]
> Upgrade your runner to at least wine 4.16 and it should go away automagically.

Thank you, it worked with newest version lutris-4.16, however I had to totally reinstall the game, it was playing sluggish after selecting the newer runner. I had to manually update the runner by the way, though it was done automatically with lutris updates.
